examples/server_profile.tf

```diff
@@ -10,83 +10,56 @@ provider "oneview" {
 data "oneview_scope" "scope" {
   name = "testing"
 }
-/*
-# Creation of Server Profile without template
-resource "oneview_server_profile" "SP" {
-  name               = "TestSP2"
-  hardware_name      = "0000A66101, bay 3"
+
+# Creates a server profile from server profile tempalte
+
+resource "oneview_server_profile" "SP2" {
+  name               = "TestSP"
+  hardware_name      = "Synergy-Encl-2, bay 8"
   type               = "ServerProfileV12"
-  enclosure_group    = "EG"
-  bios_option {
```

Comment:
any reason for removing bios config..?

Reply:
Its because i was working on the issue, maybe i missed to add them back, its updated